dev-db/pg-gvm/pg-gvm-22.4.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Greenbone Library for helper functions in PostgreSQL"
+HOMEPAGE="https://www.greenbone.net https://github.com/greenbone/pg-gvm"
+SRC_URI="https://github.com/greenbone/pg-gvm/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+# Tests requires a running database that match up with the current
+# testing slot. Won't run from ${ED}, want's to install too early.
+RESTRICT="test"
+
+DEPEND="
+	acct-group/gvm
+	acct-user/gvm
```

Comment:
Is `acct-{user,group}/gvm` really a `(R)DEPEND` of pg-gvm?

Reply:
Not sure.
Here: https://wiki.gentoo.org/wiki/Greenbone_Vulnerability_Management#PostgreSQL_backend
we ask to create the gvm database-user to be the same as the user running gvmd, and to grant all privileges to gvm on the gvmd database.
The extension has to be created on the same database. 

What (I think) the gentoo user has to do after installation is:
~$ sudo -u gvm psql gvmd
gvmd=>	CREATE EXTENSION "pg-gvm";
gvmd=>	\q